engine/src/main/resources/org/camunda/bpm/engine/db/create/activiti.db2.create.history.sql

```diff
@@ -187,6 +187,7 @@ create table ACT_HI_COMMENT (
     FULL_MSG_ BLOB,
     TENANT_ID_ varchar(64),
     REMOVAL_TIME_ timestamp,
+    REV_ integer not null default 1,
```

Comment:
Why? - https://confluence.camunda.com/display/AP/Maintaining+Backwards+Compatibility#:~:text=Add%20new%20column%20(not%20null%20but%20with%20default%20value)

Reply:
It's not that we could not add `REV_` without default value because it would be nullable and null by default, [which is backward compatible](https://confluence.camunda.com/display/AP/Maintaining+Backwards+Compatibility#:~:text=add%20new%20column%20(nullable)). But nullable does not make sense for `REV_` because we want all `Comment` entities to have a revision index.

@yanavasileva, did I capture your intention correctly?

Reply:
Several other `REV_` columns in the same file are nullable, though, and don't have a default value.